### DISCUSSION TITLE
Краткое руководство по GitHub Discussions

### DISCUSSION THREAD

**Discussion by audetv** [terratensor/svodd#78] [Вопрос-ответ (Q&A)] [answered]
Краткое руководство по GitHub Discussions. 
https://docs.github.com/ru/discussions/quickstart

**Comment by iprst** [comment]
Нужно пригласить товарищей [1](https://fct-search.audetv.ru/?search%5Bquery%5D=3a2a050ded81218f2529e3201d9096f2), [2](https://fct-search.audetv.ru/?search%5Bquery%5D=7e37f68fa5bb0ce4beea587ca5d802bb) и [3](https://fct-search.audetv.ru/?search%5Bquery%5D=3d17e527d43e5eb8b2b215ab7d99a300) — они все вносили «программную» лепту в обсуждениях. Товарищ hedvikpirs в комментах давал ссылку на свой гитхаб, но неизвестна её актуальность. Может ещё кого забыл.

**Reply by audetv** [reply]
> Товарищ hedvikpirs

Посмотрел, да, товарищ [hedvikpirs](https://fct-search.audetv.ru/?search%5Bquery%5D=hedvikpirs&search%5Bmatching%5D=&search%5Bmatching%5D=query_string) 

> обновил скрипт вот этот:
> https://github.com/hedvikpirs/apps_scrip/blob/main/code_no_Img.gs

Во всяком случае выше процитировал репозиторий существует, можно направить сообщение. Я вот только не знаю всех, существующих механизмов приглашения в дискуссии или issue.  В настройках могу отправлять приглашения на доступ к репозиторию, т.е непосредственно для работы уже для пул реквестов и т.д. 
Я вам направил pendding invite, проверим. Обычно в закрытых репозиториях я так добавляю участников, программистов, чтобы могли делать пул реквесты, так делаю в текущей работе, и в прошлых работах, но таких дискуссий ранее не вел.
Попробую упомянуть hedvikpirs в теме #62  со значком @

> Нужно пригласить товарищей [1](https://fct-search.audetv.ru/?search%5Bquery%5D=3a2a050ded81218f2529e3201d9096f2), [2](https://fct-search.audetv.ru/?search%5Bquery%5D=7e37f68fa5bb0ce4beea587ca5d802bb) и [3](https://fct-search.audetv.ru/?search%5Bquery%5D=3d17e527d43e5eb8b2b215ab7d99a300)

По товарищам 1,2,3 поддерживаю. Надо их оповестить.

**Comment by iprst** [comment]
Мне инвайт пришёл, я подтвердил. Далее постепенно соберётся народ, Юрий вон на ФКТ зовёт уже.

**Reply by audetv** [reply]
Я приглашал в проект ФКТ и вам недавно отправил приглашение на доступ к самому репозиторию, приходило сегодня приглашение?
Нужно знать или email приглашаемого и на email придет ссылка с приглашением и регистрацией, или его username в гите, т.е. он уже долен быть зарегистрирован.
Да, отправлю еще приглашение в проект товарищу hedvikpirs

**Reply by audetv** [reply, marked as answer]
Да вижу, надпись pending invite исчезла, значит все ок, значит так тоже буду отправлять приглашения.

**Comment by audetv** [comment]
Сделал update #62 
Приглашаем к обсуждения всех желающих: @hedvikpirs
